tests/e2e/healthcheck/healthcheck_test.go

```diff
@@ -266,7 +266,7 @@ func testAppHealthCheckProtocol(t *testing.T, protocol string) {
 			// Service invocation should fail
 			go func() {
 				res, status := invokeFoo(t)
-				require.Contains(t, string(res), "ERR_DIRECT_INVOKE")
+				require.Contains(t, string(res), "ERR_CONVERSATION_DIRECT_INVOKE")
```

Comment:
let's use the constant value from `errorcodes` package. the same in all tests

Reply:
I'm in the camp of always using hardcoded error codes in tests, so that it's very obvious if we break compatibility in a later version.